### PR TITLE
Fix for issue #318

### PR DESCRIPTION
**Pull request by andybellenie**

### CODE REVIEW

**Comment by andybellenie** [comment]
My first contribution so pls be gentle :) I didn't do any heavy refactoring as I'm unfamiliar with the codebase (and rust, tbh) so you may want to do some additional renaming of functions.

File: doc/build.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ As before, node 1 will create the blockchain and begin mining. As we'll be runni
 
 First, we run a wallet server to receive rewards on port 15000 (we'll log in debug mode for more information about what's happening)
```
Comment (dougyoung) [inline doc/build.md]: I think the language in the documentation should be updated as well, something like `run a wallet service to listen and receive rewards on ...`.

File: src/bin/grin.rs [hunk 2]
```diff
@@ -397,7 +397,7 @@ fn wallet_command(wallet_args: &ArgMatches) {
 		.expect("Failed to derive keychain from seed file and passphrase.");
 
 	match wallet_args.subcommand() {
-		("receive", Some(receive_args)) => if let Some(f) = receive_args.value_of("input") {
+		("listen", Some(receive_args)) => if let Some(f) = receive_args.value_of("input") {
```
Comment (dougyoung) [inline src/bin/grin.rs]: Should be `listen_args`.

**Comment by andybellenie** [comment]
Thanks, Doug, I've made the suggested changes and also incorporated graceful depreciation as suggested by @sesam

**Comment by ignopeverell** [comment]
Thanks! And thanks to @dougyoung for tackling the same issue and reviewing this PR. Keep it coming!

**Pull request merged**